scss/boost_union/post.scss

```diff
@@ -598,26 +602,30 @@ body.hasstickyfooter #back-to-top {
     /* And on smaller screens. */
     @include media-breakpoint-down(sm) {
         &.theme-block-region-outside-fullwidth,
-        &.theme-block-region-outside-coursecontentwidth {
+        &.theme-block-region-outside-coursecontentwidth,
+        &.theme-block-region-footer-fullwidth,
+        &.theme-block-region-footer-coursecontentwidth {
             /* Just show the block full width with some horizontal space. */
             padding-left: 15px;
             padding-right: 15px;
             width: 100%;
         }
-        &.theme-block-region-outside-herowidth {
+        &.theme-block-region-outside-herowidth,
+        &.theme-block-region-footer-herowidth {
             /* Remove the margin. */
             margin: 0;
         }
     }
 }
 
 /* Additional styling for content width regions to limit the block region size on Moodle pages with limited width. */
-body.limitedwidth .theme-block-region-outside-coursecontentwidth {
+body.limitedwidth .theme-block-region-outside-coursecontentwidth,
+body.limitedwidth .theme-block-region-footer-coursecontentwidth {
     max-width: $course-content-maxwidth;
 }
 
 /* Additional styling for hero-width regions to remove some styling which is counterproductive. */
-.theme-block-region-outside-herowidth {
+.theme-block-region-outside-herowidth, .theme-block-region-footer-herowidth {
```

Comment:
Would you mind to move the added selector to a separate line for better readability?